app/code/Magento/Quote/Model/QuoteManagement.php

```diff
@@ -230,6 +237,7 @@ public function __construct(
         $this->accountManagement = $accountManagement;
         $this->customerSession = $customerSession;
         $this->quoteFactory = $quoteFactory;
+        $this->checkoutHelper = $checkoutHelper;
```

Comment:
Add your property after others and add the objectManager as backward compatibility.